ci/docker/Dockerfile.build.android_armv8

```diff
@@ -73,7 +73,7 @@ ENV CXX=${CROSS_ROOT}/bin/${CROSS_TRIPLE}-clang++
 # Build ARM dependencies.
 COPY install/android_arm64_openblas.sh /work/
 RUN /work/android_arm64_openblas.sh
-ENV CPLUS_INCLUDE_PATH /work/deps/OpenBLAS
+ENV OpenBLAS_HOME=/work/deps/OpenBLAS
```

Comment:
is this change also relevant for the other arm builds?

Reply:
> Did you figure out if this affects anything downstream?

What do you mean by that? If any other image is using this docker image? I didn't check that, but it shouldn't affect any downstreams, unless they're depending on this, which would be depending on an undocumented hack.

> and is it worth moving this change to a different PR for PR atomicity, etc.

The problem is that this change is **required** for this PR to be merged. Without this change it won't work.

Reply:
Yup - I saw that. All good! My bad, I didn't read to the end before commenting =P